.kokoro/continuous/continuous.cfg

```diff
@@ -1 +1,12 @@
-# Format: //devtools/kokoro/config/proto/build.proto
\ No newline at end of file
+# Format: //devtools/kokoro/config/proto/build.proto
+
+# Disable system tests.
```

Comment:
Could we also add a configuration for system tests similar to the ones in https://github.com/googleapis/python-datastore/tree/main/.kokoro/presubmit?

It looks like currently only `system-3.12.cfg` is being used

Reply:
Do you mean changing NOX_SESSION like this?

```
env_vars: {
    key: "NOX_SESSION"
    value: "blacken doctests format system-3.12"
}
```

I think for the system configuration files to work, we'd need to make changes to google3